PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -136,6 +136,8 @@ class HomeViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.accessibilityIdentifier = "home"
+        navigationController?.delegate = self
```

Comment:
> [!NOTE]
> this was missing in the existing implementation, thus the delegate methods, which have an implementation, were not called.

Reply:
ooh nice catch! I'm surprised we didn't notice any issue prior to this PR in terms of the delegate methods not being called.